### PR TITLE
✨ SARIF 3: add flag to yaml

### PR DESCRIPTION
**Pull request by laurentsimon**
* **Please check if the PR fulfills these requirements**
- [] Tests for the changes have been added (for bug fixes / features)
- [X] PR title follows the guidelines defined in  https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
feature

* **What is the current behavior?** (You can also link to an open issue here)
no metadata

* **What is the new behavior (if this is a feature change)?**
add tags so we can use them in SARIF. (tags are optional in SARIF)

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
no

* **Other information**:

### CODE REVIEW

File: docs/checks/checks.yaml [hunk 1]
```diff
@@ -17,6 +17,10 @@
 checks:
   Active:
     risk: High
```
Comment (laurentsimon) [inline docs/checks/checks.yaml:19]: do we need some comments explaining the recall/precision?

File: docs/checks/checks.yaml [hunk 2]
```diff
@@ -61,6 +73,10 @@ checks:
         Build from source.
   Branch-Protection:
     risk: High
+    tags: supply-chain, security, source code, code review
```
Comment (oliverchang) [inline docs/checks/checks.yaml:76]: source-code, code-review (with the -) for consistency with "supply-chain"?

**Review by oliverchang — APPROVED** [review]

File: docs/checks/checks.yaml (hunk 2 above)
Comment (laurentsimon) [reply docs/checks/checks.yaml]: done.

**Review by inferno-chromium — APPROVED** [review]

**Comment by laurentsimon** [comment]
@naveensrinivasan getting
```
/bin/bash: protoc: command not found
make: *** [Makefile:93: cron/data/metadata.pb.go] Error 127
```

**Comment by naveensrinivasan** [comment]
> @naveensrinivasan getting
> 
> ```
> /bin/bash: protoc: command not found
> make: *** [Makefile:93: cron/data/metadata.pb.go] Error 127
> ```

Here is a much cleaner solution to solve this issue of generating proto files   https://github.com/tektoncd/chains/pull/192 which is also portable. I can take on this by reusing some of this code.

**Pull request merged**